pom.xml

```diff
@@ -23,7 +23,7 @@
 
     <groupId>com.rovio.ingest</groupId>
     <artifactId>rovio-ingest</artifactId>
-    <version>1.0.5_spark_3.0.1-SNAPSHOT</version>
```

Comment:
If I'm not mistaken there needs to be first a separate commit for the 1.0.5 release (not snapshot). Maybe best to do the release version commits as a separate PR anyway. Latest example: https://github.com/rovio/rovio-ingest/pull/35 (note the 2 separate commits).